### ISSUE TITLE
Missing Checkpoints

### ISSUE THREAD

**Issue by chekirou** [ucuapps/OpenGlue#4]
Hi, 
Thanks for the great implementation. I was wondering when the rest of the checkpoints were going to be released.

**Comment by marichka-dobko** [comment]
Hi, thank you for the kind comment. 
We have released all the best models for each set of local feature extractors tested and reported in our manuscript. For example, for SuperPoint we released weights for the two best OpenGlue models: one used SuperPoint with KITTI, and the other used weights trained on the COCO dataset.
For SIFT descriptors, on the other hand, you can find two models: rotation and with scale-rotation included during training. The combination of rotation and scale showed better results in our experiments, but the difference was not severe, so we decided to include both options.

**Comment by chekirou** [comment]
I am getting very bad results with the coco version of openglue. I used this configuration : 
`name:` 'SuperPointNetBn'

`descriptor_dim:` &DIM 256

`parameters:`

  `max_keypoints:` 2048

  `descriptor_dim:` *DIM

  `nms_kernel:` 9

  `remove_borders_size:` 4

  `keypoint_threshold:` 0.005

  `weights:` '/home/ostap/projects/pytorch-superpoint/logs/superpoint_coco_heat2_0/checkpoints/superPointNet_170000_checkpoint.pth.tar`

For the kitti version, the results are better but very far from the matching produced by superGlue, I used this configuration : 

`data:`
  `root_path:` '/datasets/extra_space2/ostap/MegaDepth'
  `train_list_path:` '/home/ostap/projects/superglue-lightning/assets/megadepth_train_2.0.txt'
  `val_list_path:` '/home/ostap/projects/superglue-lightning/assets/megadepth_valid_2.0.txt'
  `test_list_path:` '/home/ostap/projects/superglue-lightning/assets/megadepth_valid_2.0.txt'

  `batch_size_per_gpu:` 2
  `dataloader_workers_per_gpu:` 4
  `target_size:` [ 960, 720 ]
  `val_max_pairs_per_scene:` 50
  `train_pairs_overlap:` [0.15, 0.7]

`inference:`
  `match_threshold:` 0.2

`superglue:`
  `descriptor_dim:` &DESCRIPTOR_DIM 256
  `laf_to_sideinfo_method:` 'none'
  `positional_encoding:`
    `hidden_layers_sizes:` [32, 64, 128]
    `output_size:` *DESCRIPTOR_DIM
  `attention_gnn:`
    `num_stages:` 9
    `num_heads:` 4
    `embed_dim:` *DESCRIPTOR_DIM
    `attention:` 'softmax'
    `use_offset:` False
  `dustbin_score_init:` 1.0
  `otp:`
    `num_iters:` 20
    `reg:` 1.0
  `residual:` True



`features:`
  `name:` 'SuperPointNetBn'
  `descriptor_dim:` &DIM 256
  `parameters:`
    `max_keypoints:` 1024
    `descriptor_dim:` *DIM
    `nms_kernel:` 3
    `remove_borders_size:` 4
    `keypoint_threshold:` 0.005
    `weights:` '/home/hc/workspace/OpenGlue/experiment_1/superPointNet_50000_checkpoint.pth.tar'

Are these configurations the right ones ?
